server/eclipse-project/pom.xml

```diff
@@ -4,9 +4,12 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>us.freeandfair.production</groupId>
 	<artifactId>colorado_rla</artifactId>
-	<version>0.0.1</version>
+	<version>0.7.0-alpha</version>
```

Comment:
Makes sense.  I left in the fix for the build warnings.

We've been in 0.7.0-alpha for a week now, but I'll do it in a separate PR, after this PR lands.